troubleshooting/faq.md

```diff
@@ -161,6 +162,26 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## What should I do with git dependencies on closed networks?
+
+If you are installing the package that have a dependency on a git url, npm will fetch that package from github itself because git dependencies are valid in npm.
+
+```
+"dependencies" : {
+  "name1" : "git://github.com/user/project.git#commit-ish",
+  "name2" : "git://github.com/user/project.git#commit-ish"
+}
+```
+When you are installing the package in the closed network where a dependency references a git url rather than a package name, you are unable to access due to no internet connectivity.
```

Comment:
`you are unable to access due to no internet connectivity`

->

`you will be unable to access due to no internet connectivity`